packages/components/src/internal/components/project/ProjectLookAndFeelForm.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { TEST_PROJECT_CONTAINER } from '../../containerFixtures';
+
+import { ProjectLookAndFeelFrom } from './ProjectLookAndFeelForm';
+
+describe('ProjectLookAndFeelFrom', () => {
+    test('container datetime format', () => {
+        render(<ProjectLookAndFeelFrom container={TEST_PROJECT_CONTAINER} />);
```

Comment:
nit: Consider doing more within these tests to ensure that save works as expected, callbacks have the correct arguments, etc. The happy path.

Reply:
added